cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -1764,14 +1749,16 @@ def employee_accrual_insert(self) :
 
 
 
+#-----------------Submit Benefit Accrual from Salary Slip----------------
+
     def employee_accrual_submit(self) :  
```

Ruff (0.8.2)

Missing return type annotation for public function `employee_accrual_submit`

Add return type annotation: `None`

(ANN201)